nhais/README.md

```diff
@@ -0,0 +1,40 @@
+# NHAIS Adaptor - Developer Notes
```

Comment:
You should add information about used environment variables

Reply:
Added.